src/components/EPNSChannelItems.js

```diff
@@ -104,7 +103,7 @@ function EPNSChannelItems({ numberOfChannels }) {
           );
         })}
       </ItemH>
-      {/* <ItemH margin="40px 0px">
+      <ItemH margin="70px 0px 30px 0px">
         {secondChannels.map(function (item, id) {
```

Comment:
As the id is not available in array it is better to name it as index

Reply:
Same for here. Not sure to change the name.

Reply:
ok...